src/Faker/Provider/Image.php

```diff
@@ -28,41 +26,55 @@ class Image extends Base
      *
      * @return string
      */
-    public static function imageUrl($width = 640, $height = 480, $category = null, $randomize = true, $word = null, $gray = false)
-    {
-        $baseUrl = "https://lorempixel.com/";
-        $url = "{$width}/{$height}/";
+    public static function imageUrl(
+        int $width = 640,
```

Comment:
Could we remove the type hints on the parameters?

Reply:
You are right. If someone overloaded this method and uses it with different types this could cause issues

Reply:
Updated and kept the interface intact to prevent the BC